modules/hcp-fips.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * hosted-control-planes/hcp-prepare/hcp-requirements.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-fips_{context}"]
+= FIPS-enabled hosted clusters
+
+The binaries for {hcp} are FIPs compliant, with the exception of the {hcp} command-line interface, `hcp`.
+
+If you want to deploy a FIPS-enabled hosted cluster, you must use a FIPS-enabled management cluster. To enable FIPS mode for your management cluster, you must run the installation program from a {op-system-base-full} computer configured to operate in FIPS mode. For more information about configuring FIPS mode on {op-system-base}, see link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/security_hardening/switching-rhel-to-fips-mode_security-hardening[Switching {op-system-base} to FIPS mode].
+
+When running {op-system-base} or {op-system-first} booted in FIPS mode, {product-title} core components use the {op-system-base} cryptographic libraries that have been submitted to NIST for FIPS 140-2/140-3 Validation on only the x86_64, ppc64le, and s390x architectures.
```

Comment:
Should NIST be expanded here? I think it's National Institute of Standards and Technology as per the repo search.

Reply:
Good question! I think it's probably okay to keep as-is, as I got that language from the FIPS snippet in the openshift-docs repo: https://github.com/openshift/openshift-docs/blob/main/snippets/fips-snippet.adoc